fnplugin.md

```
@@ -0,0 +1,2035 @@
+
+BEST PRACTICE - Export queries and transforms from index:
+- Reference: Lines 423-424 show `export * from './queries/index'` and `export * from './transforms/index'`
+- Pagination drift: pagination has NO queries/ or transforms/ directories - all logic is inline
+- Issue: Pagination mixes concerns in base-pagination-plugin.ts (Lines 198-294)
+- Fix: Extract helper functions (replaceHeader, replaceFooter, ensureHeader, ensureFooter, removeByType, enforceHeaderFooterInvariants) into separate modules
+-->
+/**
+ * @file Automatically generated by barrelsby.
+ */
+
+export * from './BaseFootnoteDefinitionPlugin';
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)